projects/igniteui-angular/src/lib/grids/hierarchical-grid/hierarchical-grid.selection.spec.ts

```diff
@@ -49,7 +49,7 @@ describe('IgxHierarchicalGrid selection #hGrid', () => {
             rowIsland2 = fix.componentInstance.rowIsland2;
         }));
 
-        it('should allow only one cell to be selected in the whole hierarchical grid.', (async () => {
+        it('should allow only one cell to be selected in the whole hierarchical grid.',  () => {
```

Comment:
Changes to the hierarchical-grid.selection.spec.ts should not be included in this PR